packages/application-extension/schema/commands.json

```diff
@@ -46,6 +46,11 @@
       "command": "application:toggle-presentation-mode",
       "keys": [""],
       "selector": "body"
+    },
+    {
+      "command": "application:focusFirstElementInRightSidebar",
+      "keys": ["Alt Shift Z"],
```

Comment:
Why this shortcut? Is it standard? Would users know to use it?

Reply:
This shortcut was chosen due to its proximity to the tab key, secondly for keyboard users there is limited disruption to their hand alignment and positioning when using the shortcut.
This shortcut is not standard as the problem it solves is unique to this application (tab trap in notebooks). It also allows keyboard users to navigate the app more easily as an additional benefit